easybuild/easyconfigs/p/Panoply/Panoply-5.2.9.eb

```
@@ -0,0 +1,32 @@
+easyblock = 'PackedBinary'
+
+name = 'Panoply'
+version = '5.2.9'
+
+homepage = 'https://www.giss.nasa.gov/tools/panoply'
+description = "Panoply plots geo-referenced and other arrays from netCDF, HDF, GRIB, and other datasets."
+
+
+toolchain = SYSTEM
+
+source_urls = ['https://www.giss.nasa.gov/tools/panoply/download/']
+sources = ['%(name)sJ-%(version)s.tgz']
+checksums = ['2279c789a3ef4820ad466592b72ad840c84d427a06fa585205eec40ab3c52857']
+
+dependencies = [
+    ('Java', '11', '', SYSTEM),
+]
+
+postinstallcmds = [
+    'mkdir %(installdir)s/bin',
+    'mv %(installdir)s/panoply.sh %(installdir)s/bin',
+    'sed -i "s/jars/..\/jars/g" %(installdir)s/bin/panoply.sh',
```

To fix ` W605 invalid escape sequence '\/'`:
```suggestion
    'sed -i "s,jars/..,jars,g" %(installdir)s/bin/panoply.sh',
```